src/components/index.js

```diff
@@ -66,7 +66,7 @@ export { default as Notification } from './Notification/Notification';
 export { default as NotificationDrawer } from './NotificationDrawer/NotificationDrawer';
 export { default as Overlay } from './Overlay/Overlay';
 export { default as Profile } from './Profile/Profile';
-export { default as SearchBar } from './SearchBar/SearchBar.js';
+export { default as SearchBar } from './SearchBar/SearchBar.tsx';
```

Comment:
FYI you don't need the `.tsx` for TypeScript files

Reply:
Done. I think my vs-code auto updated that one.

Reply:
Done. I think VSC must have done this automatically when I changed the file ext.